smrf/framework/model_framework.py

```diff
@@ -263,7 +263,7 @@ def _setup_date_and_time(self):
         self.date_time = pd.date_range(
             self.start_date,
             self.end_date,
-            freq=f"{self.config['time']['time_step']}min",
+            freq="{[time][time_step]}min".format(self.config),
```

Comment:
T.I.L. about using dictionaries in string formats!

Reply:
Ditto - There are many awesome tricks in that format bag:
https://docs.python.org/3.7/library/string.html#formatstrings